src/config.ts

```diff
@@ -1,476 +1,329 @@
+);
+const ERC20_META_TXN_DOMAIN_TYPE = JSON.parse(
+  '[{ "name": "name", "type": "string" }, { "name": "version", "type": "string" }, { "name": "verifyingContract", "type": "address" }, { "name": "salt", "type": "bytes32" }]'
+);
+const CUSTOM_META_TXN_TYPE = JSON.parse(
+  '[{ "name": "nonce", "type": "uint256" }, { "name": "from", "type": "address" }, { "name": "functionSignature", "type": "bytes" }]'
+);
 
-const erc20ABIByToken: Map<string, ContractInterface> = new Map();
-erc20ABIByToken.set(tokensMap.USDC[5].address, EIP2612_ABI);
-erc20ABIByToken.set(tokensMap.USDC[1].address, EIP2612_ABI);
-erc20ABIByToken.set(tokensMap.USDC[137].address, EIP2612_ABI);
-erc20ABIByToken.set(tokensMap.USDC[43114].address, EIP2612_ABI);
+const NATIVE_ADDRESS = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee";
 
 // Set Mumbai Network Default Supported Tokens
-defaultSupportedTokens.set(80001,
-	[
-	{
-		"tokenSymbol": "USDT",
-		"decimal": 18,
-		"address": "0xeabc4b91d9375796aa4f69cc764a4ab509080a58"
-	},
-	{
-		"tokenSymbol": "USDC",
-		"decimal": 6,
-		"address": "0xda5289fcaaf71d52a80a254da614a192b693e977"
-	},
-	{
-		"tokenSymbol": "DAI",
-		"decimal": 18,
-		"address": "0x27a44456bedb94dbd59d0f0a14fe977c777fc5c3"
-	}
+const defaultSupportedTokens: Map<number, SupportedToken[]> = new Map();
+defaultSupportedTokens.set(80001, [
+  {
+    tokenSymbol: "USDT",
+    decimal: 18,
+    address: "0xeabc4b91d9375796aa4f69cc764a4ab509080a58",
```

Comment:
these could come from a .env or .yaml file. we can still do this for now but we'll have to create a follow up story to handle it